internal/envoy/manager/envoy_callbacks.go

```diff
@@ -21,6 +21,8 @@
 // SOFTWARE.
 
 // Callbacks are called by GRPC server on new events.
+//
+// `l.logger.V(1)` is effectively debug level.
```

Comment:
Why is this comment here?

Reply:
Serves as documentation.

To me, it's not clear that `c.logger.V(1).Info` is a debug level print statement.

Reply:
If V(1) returns a logger, create a variable called debugLog := l.logger.V(1) and then use that. The code can then describe what it is

Reply:
Resolved. Introduced `debuggerLogger`.